pkg/interfaces/contracts/vault/IBasePool.sol

```diff
@@ -5,14 +5,16 @@ pragma solidity ^0.8.24;
 import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 
 import { ISwapFeePercentageBounds } from "./ISwapFeePercentageBounds.sol";
+import { IUnbalancedLiquidityInvariantRatioBounds } from "./IUnbalancedLiquidityInvariantRatioBounds.sol";
 import { SwapKind, PoolSwapParams } from "./VaultTypes.sol";
 
 /**
  * @notice Base interface for a Balancer Pool.
- * @dev All pool types should implement this interface. Note that it also requires implementation of
- * `ISwapFeePercentageBounds` for all pools, to specify the minimum and maximum swap fee percentages.
+ * @dev All pool types should implement this interface. Note that it also requires implementation of:
+ * - `ISwapFeePercentageBounds` to specify the minimum and maximum swap fee percentages.
+ * - `IInvariantRatioBounds` to specify how much the invariant can change during an unbalanced liquidity operation.
```

Comment:
```suggestion
 * - `IUnbalancedLiquidityInvariantRatioBounds` to specify how much the invariant can change during an unbalanced liquidity operation.
```

This will break lint, probably it needs an extra line.